src/main/resources/db/changelog/version-1.0/28.07.2023.part1.create-address-table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE address
+(
+    id      UUID         NOT NULL PRIMARY KEY,
+    line    VARCHAR(255) NOT NULL,
+    city    VARCHAR(255) NOT NULL,
```

Comment:
please check standarts for defining length on fields. It's too much right now

Reply:
fixed